.azuredevops/components/rocSOLVER.yml

```diff
@@ -0,0 +1,49 @@
+parameters:
+- name: checkoutRepo
+  type: string
+  default: 'self'
+- name: checkoutRef
+  type: string
+  default: ''
+- name: aptPackages
+  type: object
+  default:
+    - cmake
+    - ninja-build
+    - rocblas
+    - rocsparse
+    - hipsparse
+    - rocsolver
+    - libsuitesparse-dev
+    - gfortran
+    - libfmt-dev
+    - git
```

Comment:
Add a new line in between the blocks here

Reply:
done